html-forms/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form action="#" method="post">
+      <div>
+        <p>
+          <label for="email">Email</label>
+        </p>
+        <input type="email" name="email" id="email" />
+      </div>
+
+      <div>
+        <p>
+          <label for="dropdown">Why is coding cool?</label>
+        </p>
+        <select name="dropdown" id="dropdown">
+          <option selected hidden>Select an option</option>
+          <option value="option-one">Because it's interesting</option>
+          <option value="option-two">Because it's fun</option>
+          <option value="option-three">Because it's powerful</option>
+        </select>
+      </div>
+
+      <div>
+        <p>Favorite Language (choose one)</p>
+        <input type="radio" name="radio" id="first-radio" />
+        <label for="first-radio">HTML</label>
+
+        <input type="radio" name="radio" id="second-radio" />
+        <label for="second-radio">CSS</label>
+
+        <input type="radio" name="radio" id="third-radio" />
```

Comment:
These `input` tags should also have a `value` attribute with corresponding values (for the same reason as the `option` tag above.